docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -134,7 +134,6 @@ when using security-sensitive API modules in mobile native applications.
 | `http://tizen.org/privilege/packagemanager.admin` | platform | -            | 2.3   | The application can install and uninstall application packages. |
```

Comment:
Line 118: The application can determine the user's approximate location including the user device's Cell ID, LAC (Location Area Code), and TAC (Tracking Area Code).

Please change:
Location Area Code (LAC) and Tracking Area Code (TAC)